addons/website_event_track/models/event_track.py

```diff
@@ -51,30 +51,42 @@ def _get_default_stage_id(self):
              " * Grey is the default situation\n"
              " * Red indicates something is preventing the progress of this track\n"
              " * Green indicates the track is ready to be pulled to the next stage")
-    # speaker
-    partner_id = fields.Many2one('res.partner', 'Speaker')
+    partner_id = fields.Many2one('res.partner', 'Contact', help="Contact information of the speaker.")
```

Comment:
Not sure help is completely correct. As far as I understand it, partner_id is more "contact of the track that may be different from the speaker", isn't it ?